roles/network_plugin/calico/tasks/main.yml

```diff
@@ -22,16 +22,15 @@
   changed_when: false
   notify: restart calico-node
 
-- name: Calico | Determine hyperkube cni to use depending of the version of kube
+- name: Calico | Do not use hyperkube cni if kube_version under v1.3.4
```

Comment:
LGTM. Later we should remove this logic and just use the var perhaps. Because in v1.4.0 there is no more calico and logic becomes too much complicated.